lib/board.rb

```diff
@@ -1,14 +1,11 @@
 class Game_board
   attr_accessor :grid_key
-  def initialize(move = {})
-    @grid_key = move.fetch(:grid_key, grid_default)
-  end
 
-  def grid_default
-    Array.new(3){Array.new(3){"|   | "}}
+  def initialize
+     @grid_key = {}.fetch(:grid_key, Array.new(3){Array.new(3){"|   | "}})
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
- [ ] You have `print/puts` on line `35`. Move that to the `bin/main` `dir`

Reply:
* Moved the display_grid function to main, since it presents puts / print to draw the grid.
* Changed the definition of the board, to display numbers for testing purposes. Other UI changes will be done in Milestone 4 , since we are encourage there to improve whatever we want for our game.

There should be no more print / put / get in any other classes. **You'll find however a few constant strings that are for checking conditionals**, but **they are not being entered by user nor being displayed anywhere.**